app/types/next-auth.d.ts

```diff
@@ -58,7 +58,8 @@ declare module 'next-auth' {
     sendTaskEmails: boolean;
 
     cancelPrivateCloudRequest: boolean;
-    cancelPublicCloudRequest: boolean;
+    cancelPublicCloudRequestAsOwner: boolean;
```

Comment:
I do not think we will need separation of `AsOwner` and `AdAdmin`, this is global level permission, so should be admin's privilege